assemble-and-restart

```diff
@@ -50,7 +50,7 @@ fi
 # We now run the assembly script. If there is a custom one written in the
 # source files, we use that instead.
 if [ -f ${ODO_S2I_SRC_BIN_PATH}/.s2i/bin/assemble ]; then
-    ${ODO_S2I_SRC_BIN_PATH}/.s2i/bin/assemble
+    ${ODO_S2I_SRC_BIN_PATH}/src/.s2i/bin/assemble
```

Comment:
this doesn't look right :-( we are testing different path than what we are executing `/src/`.
If `${ODO_S2I_SRC_BIN_PATH}/src/.s2i/bin/assemble` is correct path, it should be updated also in if condition.

Reply:
aah that's correct...
Fixing it